protos/participant_transaction.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message CreateParticipantAction {
+    
+	// The public key associated with the participant used to sign
+	// transactions
+	string public_key = 1;
+
+	// Username of the participant
+	string username = 2;
+
+	// The ticker symbol of the participant's organization
+	string organization_id = 3;
+}
+
+message UpdateParticipantAction {
+    
+	// The public key associated with the participant used to sign
+	// transactions
+	string public_key = 1;
```

Comment:
Would  participants be able to update information on other participants?  If not, I don't think we need the public_key here, as a participant would only be able to update their own info.